asapdiscovery-simulation/asapdiscovery/simulation/simulate.py

```diff
@@ -235,6 +235,20 @@ def check_steps(cls, values):
             )
         return values
 
+    @property
+    def n_frames(self) -> int:
+        return self.num_steps // self.reporting_interval
+
+    @property
+    def total_simulation_time(self) -> openmm.unit.quantity.Quantity:
+        return self.num_steps * self.timestep * unit.femtoseconds
```

Comment:
Would be great to have units but until then we should update the field strings on the `VanillaMDSimulator` class to detail the intended units.